Dockerfile

```diff
@@ -1,12 +1,14 @@
 # Official lightweight Node.js image
 # https://hub.docker.com/_/node
-FROM node:15-slim
+FROM node:17-slim
```

Comment:
I think we should use Node 16, since that's Active LTS. Node 17 end of life is in only about a month (https://nodejs.org/en/about/releases/).

Reply:
I get an odd puppeteer error with Node 16, but Node 18 works, it'll be in LTS soon, and it's supported through 2025

Reply:
Feels like we should run our tests with the same node version that we run in production. I guess we could bump our github actions here to 18 as well.

Reply:
Thoughts?

Reply:
+1, bumped all github actions to node 18, let's see how it goes